test/accumulate_by_key-spec.js

```diff
@@ -221,3 +227,52 @@ describe('accumulate_by_key should', () => {
         expect(results.length).toBe(0);
     });
 });
+
+describe('accumulate should', () => {
+    const job = newTestJobConfig();
+
+    job.operations = [
+        {
+            _op: 'test-reader',
+            fetcher_data_file_path: path.join(__dirname, 'fixtures', 'data.json')
+        },
+        {
+            _op: 'accumulate_by_key',
+            key_field: 'id',
+            empty_after: 10,
+            flush_data_on_shutdown: true
+        }
+    ];
+
+    it('return accumulated data on flush event', async () => {
+        const testSlice = newTestSlice();
+
+        const harness = new WorkerTestHarness(job, { assetDir: path.join(__dirname, '..', 'asset') });
```

Comment:
FYI, if you don't specify the assetDir it will automatically try and it, so you would only need to specify this if it was different was in a different folder than `<projectRoot>/asset`

Reply:
cool